src/modules/tests/test_data/__tests__/test-data.reducer.spec.ts

```diff
@@ -1,42 +1,70 @@
 import { testDataReducer, initialState } from '../test-data.reducer';
-import { AddDrivingFault } from '../test-data.actions';
+import { AddDrivingFault, AddSeriousFault } from '../test-data.actions';
 import { Competencies } from '../test-data.constants';
 import { TestData } from '@dvsa/mes-test-schema/categories/B';
 
 describe('TestDataReducer reducer', () => {
-  it('should add a driving fault when no driving faults exist', () => {
-    const result = testDataReducer(initialState, new AddDrivingFault({
-      competency: Competencies.controlsGears,
-      newFaultCount: 1,
-    }));
-    expect(result.drivingFaults.controlsGears).toBe(1);
-  });
-  it('should update a driving fault when it already exists', () => {
-    const state: TestData = {
-      drivingFaults: {
-        controlsGears: 1,
-      },
-    };
+  describe('ADD_DRIVING_FAULT', () => {
+    it('should add a driving fault when no driving faults exist', () => {
+      const result = testDataReducer(initialState, new AddDrivingFault({
+        competency: Competencies.controlsGears,
+        newFaultCount: 1,
+      }));
+      expect(result.drivingFaults.controlsGears).toBe(1);
+    });
+    it('should update a driving fault when it already exists', () => {
+      const state: TestData = {
+        drivingFaults: {
+          controlsGears: 1,
+        },
+      };
 
-    const result = testDataReducer(state, new AddDrivingFault({
-      competency: Competencies.controlsGears,
-      newFaultCount: 2,
-    }));
-    expect(result.drivingFaults.controlsGears).toBe(2);
-  });
-  it('should not remove an exisitng driving fault when a new one is added', () => {
-    const state: TestData = {
-      drivingFaults: {
-        controlsParkingBrake: 1,
-      },
-    };
+      const result = testDataReducer(state, new AddDrivingFault({
+        competency: Competencies.controlsGears,
+        newFaultCount: 2,
+      }));
+      expect(result.drivingFaults.controlsGears).toBe(2);
+    });
+    it('should not remove an exisiting driving fault when a new one is added', () => {
```

Comment:
exisiting should be existing.. but obviously very very very minor

Reply:
If the only thing wrong with the pr is a typo, I'm a happy guy